src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -296,6 +307,8 @@ class ov::genai::VLMPipeline::VLMPipelineImpl {
         m_language.reset_state();
         // clear all chat history
         m_inputs_embedder->finish_chat();
+        // clear sampler requests info
+        m_sampler.clear_request_info(0);
```

Comment:
It's already cleared after `get_lm_encoded_results()`. Why is it cleared again?

Reply:
removed